docs/govuk-verify.md

```diff
@@ -35,53 +35,190 @@ curl localhost:50300/admin/healthcheck
 
 ## Managing Certificates for the IDAP PKI
 
-### Local develoment
+Two certificates are used one for signing and one for encryption.
 
-In local development mode, nothing needs to be done here.
+The are separate certificates for non-production (integration) and production
+environments
+
+### Local development
+
+In local development mode, nothing needs to be done.
 
 ### Live environments
 
+Integration certificates expire after 2 years, while production certificates
+expire after 6 months. See the
+[Verify docs for how to rotate keys](https://www.docs.verify.service.gov.uk/maintain-your-connection/rotate-keys/)
+for more information.
+
+Pre-requisites:
+
+- You are a requester, see this page on
+  [confluence](https://dfedigital.atlassian.net/wiki/spaces/TP/pages/1116110992/Verify+Service+Provider)
+- You have access to the Azure management portal for the project
+- You have the
+  [Azure cli installed](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+- You have the code-base locally
+
 #### Requesting certificates
 
 For non-production (integration) environments (`development`) and production
 environments (`production`), we need to request certificates from Verify. The
-following will guide you though the process of generating and enrolling
-certificate requests.
+following will guide you though the process of generating, enrolling and
+rotating certificates.
+
+##### Create certificate signing requests (CSR)
+
+Complete these steps for both signing and encryption certificates.
+
+Access the Azure portal and locate the key vault for the environment you are
+updating the certificates for
+
+https://azure.microsoft.com/
+
+Note the version number of the latest set of certificates, keys, CSRs and
+challenge phrase e.g. TeacherPayments`ENV`VspSaml`TYPEN`, where:
+
+- `ENV` is the environment (either Dev or Prod)
+- `TYPE` is the type of certificate (Signing or Encryption)
+- `N` is the version number
+
+Run the request script with the next version number in the sequence:
 
 ```bash
 bin/request-vsp-certs <environment> <version>
 ```
 
-Where `<environment>` is one of `development` or `production` and `<version>` is
-an integer, usually 1 higher than the version of the certificates currently in
-use.
+Generate a strong challenge phrase and keep a note of it as you will need it
+during enrolment (see below), the challenge phrase should contain only
+alphabetical or numerical characters. The following special characters are also
+allowed `' ( ) + , - . / ,` and `space`.
```

Comment:
Can we seperarte out each character in its own backtick? I stumbled over this a bit